projects/bp-strapi/unit-tests/app.test.js

```diff
@@ -0,0 +1,3 @@
+it('strapi is defined', () => {
+  expect(strapi).toBeDefined();
```

Comment:
Why is 'strapi' used, instead of 'instance'?

Reply:
We can use `strapi` as its global after the initialization. The purpose of using `instance` is just to make sure, we correctly setup strapi (and make it global) _before_ executing the tests and cleaning up the strapi service (and cleaning up the db) _after_ executing all tests.
Thats how far my understanding goes, actually we just went with this approach as it was suggested in the documentation.

I will let this comment open, as apart from the setup/cleanup we should indeed use `strapi` over `instance`.